README.md

```diff
@@ -30,11 +30,28 @@ git clone https://github.cerner.com/terra/terra-ui.com.git
 bundle install && npm install
 ```
 
+<details>
+<summary>If bundle install fails</summary>
+An error occurred while installing pg (0.20.0), and Bundler cannot continue.Make sure that gem install pg -v '0.20.0' succeeds before bundling., then install posgresql and try again.
```

Comment:
spelling error on `posgresql`
If you want to be fancy you can put alot of this in `bin/setup`
```
#!/bin/sh

# Set up Ruby dependencies
bundle install

# Ensure npm is installed
platform=`uname`
if [[ "$platform" == *"Darwin"* ]]; then
  echo "I have detected that you are on a mac! I'll assume you are using brew."
  brew install nodejs
  brew install postgresql
  brew services start postgresql
fi

npm install
```